src/vm/proftoeeinterfaceimpl.cpp

```diff
@@ -2595,21 +2599,16 @@ HRESULT ProfToEEInterfaceImpl::GetCodeInfo3(FunctionID functionId,
             CodeVersionManager* pCodeVersionManager = pMethodDesc->GetCodeVersionManager();
```

Comment:
I notice a pre-existing bug - this code path didn't grab the code version manager lock and it needs to

Reply:
If you run the tests with a debug build it should give you asserts anywhere you are missing the lock.

Reply:
Thanks. I haven't run the existing profiler tests yet, but in general I am running with debug coreclr so I should catch these once I do.